src/Spatial/Projective/Vector3DHomogeneous.cs

```diff
@@ -51,12 +51,15 @@ public Vector3DHomogeneous(double x, double y, double z, double w)
         /// </summary>
         /// <param name="vector">A mathnet.numerics vector</param>
         private Vector3DHomogeneous(Vector<double> vector)
-            : this(vector.At(0), vector.At(1), vector.At(2), vector.At(2))
         {
             if (vector.Count != 4)
             {
                 throw new ArgumentException("Size must be 4");
             }
+            this.X = vector.At(0);
```

Comment:
Please integrate master into this branch, and remove all ```this.``` qualifiers from the file.